pptx/slide.py

```diff
@@ -445,6 +494,32 @@ def remove(self, slide_layout):
         # --including images (not used elsewhere) and hyperlinks
         slide_layout.slide_master.part.drop_rel(target_sldLayoutId.rId)
 
+    def add(self, slide_layout):
```

Comment:
Where is this being used?

Reply:
Not related to the duplicate, more utility since it's a module for general use